runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -2249,8 +2249,9 @@ private ReadWithIds(UnboundedSource<T, ?> source) {
       @Override
       public void populateDisplayData(DisplayData.Builder builder) {
         super.populateDisplayData(builder);
-        builder.add(DisplayData.item("source", source.getClass()));
-        builder.include(source);
+        builder
+            .add(DisplayData.item("source", source.getClass()))
+            .include("source", source);
```

Comment:
Sounds good. I was thinking that this would be help by allowing us to use the `nested()` builder for the further customization, so this should be helpful for the future.